### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 4 - Esercizio 1

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#535] [Esercizi M-Z]
![es1](https://github.com/user-attachments/assets/4d35ea34-595c-4f89-bbff-b6bee1addde9)

**Comment by marcoseverini** [comment]
(i)
![immagine](https://github.com/user-attachments/assets/4105ce6a-ae6f-4e06-b08e-4fa5970cad72)
(ii)
![immagine](https://github.com/user-attachments/assets/3eafe0c7-e2a9-421e-8174-ab884186ac9f)
![immagine](https://github.com/user-attachments/assets/f3455057-9151-4538-a27a-029034e981af)
(iii)
![immagine](https://github.com/user-attachments/assets/f391fc54-9c3a-49c7-b2e8-0ea4a50c4cd2)
(iv)
![immagine](https://github.com/user-attachments/assets/1849350b-6518-42f1-9ab6-c80761290bc0)
